tests/test_reason/test_standardizederror.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+from doubtlab.reason import StandardizedErrorReason
+
+
+def test_from_predict():
+    """Test `from_predict` on an obvious examples"""
+    y = np.array([0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9])
+    preds = np.array(
+        [0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 10.0]
+    ) + np.random.choice([-0.05, 0, 0.05], 10)
+    predicate = StandardizedErrorReason.from_predict(pred=preds, y=y, threshold=3.0)
+    assert np.all(
+        predicate == np.array([0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0])
+    )
+
+    y = np.array([0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9])
```

Comment:
Bit of a nitpick, but I think there's no reason to re-define `y` here. If you redefined it to make it clear that you're testing another instance it may be better to add another test instead (maybe something like `test_from_predict_no_reason`).

Reply:
Indeed the idea was to have two different tests with kind of obvious results:
- one in which the error of one prediction is quite large
- one where nothing should be detected because all predictions are close to the true value

I am splitting into two test functions.

Would you suggest to test on something different?

Reply:
I think this is fine 👍